lib/domain/entities/user.dart

```diff
@@ -0,0 +1,52 @@
+import 'package:equatable/equatable.dart';
+
+class User extends Equatable {
+  final String firstName;
```

Comment:
Feature Suggestion: 
Its even better if we also store social handles like twitter and instagram or personal blog(hashnode etc.) of the users and show them in their profile.

Reply:
Yup @Harryalways317, thats' a great one. I actually thought of taking in one handle but we updated it.